docker/entrypoint.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -e
+
+if [ "$1" = 'celestia-da' ]; then
+    echo "Initializing Celestia Node with command:"
+
+    if [[ -n "$NODE_STORE" ]]; then
+        echo "celestia-da "${NODE_TYPE}" init --p2p.network "${P2P_NETWORK}" --node.store "${NODE_STORE}""
+        celestia-da "${NODE_TYPE}" init --p2p.network "${P2P_NETWORK}" --node.store "${NODE_STORE}"
```

Comment:
where do you pass the auth token?

Reply:
It's no longer required once #19 is merged. It defaults to the `ReadWrite` permissions token autogenerated or read from the node store. It can be overridden with `--da.grpc.token` but we don't need to pass it here as the default is sufficient.